test/db/archos/linux-x64/dbg_step_back

```diff
@@ -14,7 +14,7 @@ dk 9
 EOF
 EXPECT=<<EOF
 0x00000001
-0x00000001
+0x736c6974756572
```

Comment:
checked this on my machine, fedora 32, and this value is 0. So i assume it depends on the system. The other values remain the same. Maybe a better test would be to just check rbx, r12 and rip.